src/StyleSheet.js

```diff
@@ -129,8 +129,14 @@ export default class StyleSheet {
       if (stringified[rule.id]) {
         continue
       }
+
+      if (rule.style && Object.keys(rule.style).length === 0) {
```

Comment:
As not every rule has a style property.

lets create a util isEmptyObject (separate file, same like clone)

http://jsperf.com/object-keys-length-vs-for-in-return
```
function isEmptyObject(obj) {
  for (var key in obj) return false
  return true
}
```